src/libs/logging/src/lib.rs

```diff
@@ -126,6 +174,37 @@ pub fn slog_level_to_level_name(level: slog::Level) -> Result<&'static str, &'st
     Err("invalid slog level")
 }
 
+pub fn regist_component_logger(component_name: &str) {
+    let component = String::from(component_name);
+    LOGGERS.rcu(|inner| {
+        let mut updated_inner = HashMap::new();
+        updated_inner.clone_from(inner);
+        updated_inner.insert(
+            component_name.to_string(),
+            slog_scope::logger()
+                .new(slog::o!("component" => component.clone(), "subsystem" => component.clone())),
+        );
+        updated_inner
+    });
+}
+
+pub fn regist_subsystem_logger(subsystem_name: &str) {
+    let subsystem = String::from(subsystem_name);
+    LOGGERS.rcu(|inner| {
+        let mut updated_inner = HashMap::new();
+        updated_inner.clone_from(inner);
+        updated_inner.insert(
+            subsystem_name.to_string(),
+            // This will update the original `subsystem` field.
+            inner
+                .get("runtimes")
```

Comment:
Can this 'runtimes' be a parameter to this function to be passed in? By this way, we can create subsystem logger of agent&hypervisor if needed